### DISCUSSION TITLE
AOSP 12.0 v400.b

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2059] [General]
This is a pre-release. This is not considered daily-drivable, though your mileage may vary.
Gapps variant will come later. As you'll notice, there is no A-only variant at the moment. It is still undecided whether A-only variants will later appear.

When you report issues in the bug tracker, don't forget to mention which GSI version you're using.

If you want unshared_blocks/read-write system, use vndklite variants. Non-vndklite devices are capable of running vndklite variants.

Known bugs:
- On various Mali devices (Mediatek, Samsung), there are graphical glitches at many places. You can workaround this issue by setting debug.sf.latch_unsignaled property to 0 with slower performances
- Moto One Action doesn't boot
- VoLTE/IMS is broken on Mediatek devices
- su doesn't work from apps (easy to fix manually with cp /system/xbin/su /system/bin/su)
- Alternate audio policy breaks volume on Samsung devices
- Pixel 1/2  have no sound in-call

Changes compared to v400.a:
- Fix modem for Pixel 1/Pixel 2
- November security patch level

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v400.b'>AOSP 12.0 v400.b</a>.</em>

**Comment by matifali** [comment]
What what is the difference between floss and vanilla builds?

**Reply by KnoyanMitsu** [reply]
Vanilla= no gapps and microg full minimal GSI
Floss= gapps but open soucre aka microg and google play using open soucre store (Aurora store)
